### PR TITLE
Fixed username detection on some environments

### PR DESCRIPTION
**Pull request by pallas1**

### CODE REVIEW

**Comment by pallas1** [comment]
I can't merge this, no permissions.

File: configure.sh [hunk 1]
```diff
@@ -280,9 +280,7 @@ while [[ $# -gt 0 ]]; do
     --easy)
         cert_pass="1234"
         cert_cn="LTHNEasyDeploy"
-        if [ -n "$USER" ]; then
-            LTHN_USER="$USER"
-        fi
+        LTHN_USER="`whoami`"
```
Comment (valiant1x) [inline configure.sh]: Is this an intentional change? it looks like it removes the ability to pass the script a $USER as an arg.

File: install.sh [hunk 2]
```diff
@@ -7,10 +7,10 @@ if [ -z "$CLIENT" ] && [ -z "$SERVER" ]; then
     CLIENT=y
 fi
 
-if [ "$USER" = "root" ]; then
+if [ "`whoami`" = "root" ]; then
```
Comment (valiant1x) [inline install.sh]: Again, not sure if this is intentional, but it seems to remove the ability to specify $USER as arg. [Some scripts](https://github.com/LetheanMovement/lethean-vpn/blob/master/server/easy-deploy-node.sh) use $USER

File: configure.sh (hunk 1 above)
Comment (valiant1x) [reply configure.sh]: @pallas1

Comment (pallas1) [reply configure.sh]: The $USER environment variable should contain the current user name, but it doesn't work on some environments. The best alternative I've found was replacing it with the much more compatible "whoami" call. If that variable is used to pass a username, that should be changed as you are not supposed to change it.

Comment (valiant1x) [reply configure.sh]: OK! Maybe I misunderstood the code. I thought I remembered somewhere along the line seeing a script initialized with `--runas $user`. I reviewed the code again and it looks like there is a `--runas-user` option in configure.sh but these changes *shouldn't* conflict.

**Pull request merged**